data/campaigns/Eastern_Invasion/scenarios/16_Eleventh_Hour.cfg

```diff
@@ -0,0 +1,2237 @@
+                        [/message]
+                        [message]
+                            speaker=Dacyn
+                            message= _ "I will not put the fate of all Irdya in the hands of a vengeful clansman and Weldyn’s tattered armies. I must face Mal-Ravanal in a duel, and I must be the one to strike the infernal lich down! I will trust none other than myself with such responsibility."
+                        [/message]
+                        [message]
+                            speaker=Konrad
+                            message= _ "Both of you alike speak wisdom. This is a difficult decision, and I have little time in which to make it."
+                        [/message]
+                        [message]
+                            speaker=Konrad
+                            message= _ "Lieutenant Gweddry, you are familiar with the situation, and you know both Owaec and Dacyn well. Whose plan do you suggest I follow?"
+                        [/message]
+                        [message]
+                            speaker=Gweddry
+                            message= _ "I think we should..."
+                            [option]
+                                label= _ "challenge Mal-Ravanal to a duel. <i>(normal)</i>"
+                                [command]
+                                    [message]
+                                        speaker=Konrad
+                                        message= _ "I agree. Dacyn, send the message at once."
+                                    [/message]
+                                    [endlevel]
+                                        result=victory
+                                        next_scenario=17a_The_Duel
+                                        {NEW_GOLD_CARRYOVER 0} # no need for extra gold here
+                                    [/endlevel]
+                                [/command]
+                            [/option]
+                            [option]
+                                label= _ "defeat Mal-Ravanal in pitched battle. <i>(challenging)</i>"
+                                [command]
+                                    [message]
+                                        speaker=Konrad
+                                        message= _ "I agree. Owaec, begin preparing our armies."
+                                    [/message]
+                                    [endlevel]
+                                        result=victory
+                                        next_scenario=17b_All-In
+                                        {NEW_GOLD_CARRYOVER 40}
+                                    [/endlevel]
+                                [/command]
+                            [/option]
+                        [/message]
+                    [/then]
+
+                    #--------------------
+                    # OWAEC ALIVE, GWEDDRY DEAD
```

Comment:
The first three `[message]`s seem to be the same between 'everyone is alive' and 'owaec alive gweddry dead' scenarios. Could they not be moved outside of the 'if have unit gweddry' condition?